sources/organizations/ui/src/info/Investigation.tsx

```diff
@@ -20,7 +20,19 @@ export async function Investigation(props: Props) {
     await hx_urls.organizations.leaders.$get({
       query: { siret: organization.siret },
     });
-
+  const siren = organization.siret.substring(0, 9);
+  const banaticUrl = `https://www.banatic.interieur.gouv.fr/intercommunalite/${siren}`;
+  let isBanaticAvailable = false;
+  try {
+    const response = await fetch(banaticUrl, {
+      method: "HEAD",
+      signal: AbortSignal.timeout(3000),
```

Comment:
suggestion: replace magic number to constants.
Like pci we can add `HTTP_CLIENT_TIMEOUT` to the `core/src/config/env.ts`

```suggestion
      signal: AbortSignal.timeout(HTTP_CLIENT_TIMEOUT),
```